iac/terraform/rds-elastic-search.tf

```diff
@@ -0,0 +1,121 @@
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "2.77.0"
+
+  name                 = "ecomm"
+  cidr                 = "10.0.0.0/16"
+  azs                  = ["ap-southeast-2a", "ap-southeast-2b","ap-southeast-2c"]
+  public_subnets       = ["10.0.4.0/24", "10.0.5.0/24", "10.0.6.0/24"]
+  enable_dns_hostnames = true
+  enable_dns_support   = true
+}
+resource "aws_db_subnet_group" "ecommdb" {
+  name       = "ecomm"
+  subnet_ids = module.vpc.public_subnets
+
+  tags = {
+    Name = "Ecomm"
+  }
+}
+resource "aws_security_group" "rds" {
+  name   = "ecomm_rds"
+  vpc_id = module.vpc.vpc_id
+
+  ingress {
+    from_port   = 5432
+    to_port     = 5432
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  egress {
+    from_port   = 5432
+    to_port     = 5432
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  tags = {
+    Name = "ecomm_rds"
+  }
+}
+
+resource "aws_db_instance" "ecommdb" {
+    identifier = "ecommdb"
+    allocated_storage = 10
+    db_name = "ecommdb"
+    engine = "postgres"
+    engine_version = "15.4"
+    instance_class = "db.t3.micro"
+    username="masteruser"
+    password="ecommdbpw"//change pw
+    db_subnet_group_name = aws_db_subnet_group.ecommdb.name
+    vpc_security_group_ids = [aws_security_group.rds.id]
+    parameter_group_name = "default.postgres15"
+    publicly_accessible = true
```

Comment:
This is OK for development but in a real production system you should almost never allow a database to be publicly accessible. All access should be through the app you're developing and its APIs, or the AWS console for administration.